examples/asr/emformer_rnnt/tedlium3/lightning.py

```diff
@@ -46,20 +29,21 @@ def __init__(self, base_dataset, max_token_limit):
         idx_target_lengths = [
             (idx, self._target_length(fileid, line)) for idx, (fileid, line) in enumerate(self.base_dataset._filelist)
         ]
+        idx_target_lengths = [(idx, length) for idx, length in idx_target_lengths if length != -1]
 
         assert len(idx_target_lengths) > 0
 
         idx_target_lengths = sorted(idx_target_lengths, key=lambda x: x[1])
 
         assert max_token_limit >= idx_target_lengths[-1][1]
 
-        self.batches = _batch_by_token_count(idx_target_lengths, max_token_limit)
+        self.batches = batch_by_token_count(idx_target_lengths, max_token_limit)[:100]
```

Comment:
Why we choose the first 100 batches here?

Reply:
good call — this was meant to speed up the training epoch for testing purposes only and should be removed